_rules/SC1-3-5-autocomplete-valid.md

```diff
@@ -0,0 +1,177 @@
+
+### Exepctation 3
+
+The [correct autocomplete field](#correct-autocomplete-field) is an [appropriate field for the form control](#appropriate-field-for-the-form-control).
+
+## Assumptions
+
+For this rule, it is assumed that the `autocomplete` attribute is not used on form fields that do not correspond to a autocomplete field described in the HTML 5.2 specification. If the `autocomplete` field is used to describe "custom" taxonomy, rather than that described in the specification, this rule may produce incorrect results.
+
+## Accessibility support
+
+While `autocomplete` in a promising technique for supporting personalisation in HTML, support for this is fairly limited. 
+
+## Background
+
+The intent of this rule is to ensure that the `autocomplete` attribute can be used to suport personalization. Many users may find it easier to fill out forms if those can be styled or layed out in a way that is familiar to them. Assistive technologies can do this when a form control is marked up in such a way that its purpose can be understood. For instance, assistive technologies could add familiar icons and colors to different fields, making it easier for the user to understand what the form does. 
+
+- [https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html](https://www.w3.org/WAI/WCAG21/Understanding/identify-input-purpose.html)
+- [https://www.w3.org/TR/WCAG21/#dfn-programmatically-determinable](https://www.w3.org/TR/WCAG21/#dfn-programmatically-determinable)
+- [https://www.w3.org/TR/html52/sec-forms.html#sec-autofill](https://www.w3.org/TR/html52/sec-forms.html#sec-autofill)
+
+## Test Cases
+
+### Passed
+
+```html
+<!-- single autocomplete term -->
+<input autocomplete="username" />
+```
+
+```html
+<!-- single autocomplete term for select -->
+<select autocomplete="bday-month">
+  <option>January</option>
+  <option>...</option>
+</select>
+```
+
+```html
+<!-- Autocomplete term, only valid for textarea -->
+<textarea autocomplete="Street-Address"></textarea>
+```
+
+```html
+<!-- Two autocomplete terms -->
+<input autocomplete="Work EMail" />
+```
+
+```html
+<!-- Autocomplete using section-* -->
+<input autocomplete="section-partner email" />
+```
+
+```html
+<!-- Tripple autocomplete terms -->
+<input type="text" autocomplete="section-primary billing street-address />
+```
+
+```html
+<!-- Full length autocomplete terms -->
+<input autocomplete="section-primary shipping work email" />
+```
+
+### Failed
+
+```html
+<!-- Unknown autocomplete term -->
+<input autocomplete="badterm" />
+```
+
+```html
+<!-- "work" not allowed before "photo" -->
+<input autocomplete="work photo" />
+```
+
+```html
+<!-- invalid order of terms -->
+<input autocomplete="work shipping email" />
+```
+
+```html
+<!-- comma seperated rather than space separated list -->
+<input autocomplete="work,email" />
+```
+
+```html
+<!-- inappropriate field type -->
```

Comment:
I found "Inappropriate field type" a bit ambiguous... It looks like the autocomplete attribute value is inappropriate for the input type. Is that what is going on here?